### PR TITLE
Add dashboard views for some buttons

### LINKED ISSUE #11: Add functionalities for Dashboard
Implement the layouts to the buttons, and the related CRUD to it.

Currently, CRUD for Skills and Traits are done. 

Left with Current Tasks and Task History

### PR DESCRIPTION
**Pull request by chrisjwelly**
Part of #11. Adds the views for TaskHistory, Skills and Traits without any form of functionalities. That is, it is "just for show" and you can't really do any form of creating, updating or deleting.

These are currently using the stubs in `Model.kt`.

### CODE REVIEW

File: app/src/main/java/com/zhengcode/detask/DashboardCurrentTasksActivity.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+
+class DashboardCurrentTasksActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_dashboard_current_tasks)
+    }
+}
```
Comment (chrisjwelly) [inline app/src/main/java/com/zhengcode/detask/DashboardCurrentTasksActivity.kt]:
```suggestion
}

```

File: app/src/main/java/com/zhengcode/detask/DashboardEditProfileActivity.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+
+class DashboardEditProfileActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_dashboard_edit_profile)
+    }
+}
```
Comment (chrisjwelly) [inline app/src/main/java/com/zhengcode/detask/DashboardEditProfileActivity.kt]:
```suggestion
}

```

File: app/src/main/java/com/zhengcode/detask/DashboardSettingsActivity.kt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+
+class DashboardSettingsActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_dashboard_settings)
+    }
+}
```
Comment (chrisjwelly) [inline app/src/main/java/com/zhengcode/detask/DashboardSettingsActivity.kt]:
```suggestion
}

```

File: app/src/main/java/com/zhengcode/detask/DashboardSkillsActivity.kt [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+        // when this Activity is created, check its corresponding menuItem
+        val menuItem: MenuItem = navView.menu.getItem(0)
+        menuItem.isChecked = true
+
+        navView.setOnNavigationItemSelectedListener(onNavigationItemSelectedListener)
+
+        setupRecycleView()
+    }
+}
```
Comment (chrisjwelly) [inline app/src/main/java/com/zhengcode/detask/DashboardSkillsActivity.kt]:
```suggestion
}

```

File: app/src/main/java/com/zhengcode/detask/DashboardSkillsAdapter.kt [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        }
+
+        fun setData(skillStub: SkillStub, pos: Int) {
+            itemView.txvTitle.text = skillStub.name
+
+            this.currentSkillStub = skillStub
+            this.currentPosition = pos
+        }
+    }
+}
```
Comment (chrisjwelly) [inline app/src/main/java/com/zhengcode/detask/DashboardSkillsAdapter.kt]:
```suggestion
}

```

File: app/src/main/java/com/zhengcode/detask/DashboardTaskHistoryActivity.kt [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+        // when this Activity is created, check its corresponding menuItem
+        val menuItem: MenuItem = navView.menu.getItem(0)
+        menuItem.isChecked = true
+
+        navView.setOnNavigationItemSelectedListener(onNavigationItemSelectedListener)
+
+        setupRecycleView()
+    }
+}
```
Comment (chrisjwelly) [inline app/src/main/java/com/zhengcode/detask/DashboardTaskHistoryActivity.kt]:
```suggestion
}

```

File: app/src/main/java/com/zhengcode/detask/DashboardTaskHistoryAdapter.kt [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+        }
+
+        fun setData(task: Task, pos: Int) {
+            itemView.txvTitle.text = task.name
+
+            this.currentTask = task
+            this.currentPosition = pos
+        }
+    }
+}
```
Comment (chrisjwelly) [inline app/src/main/java/com/zhengcode/detask/DashboardTaskHistoryAdapter.kt]:
```suggestion
}

```

File: app/src/main/java/com/zhengcode/detask/DashboardTraitsActivity.kt [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+        // when this Activity is created, check its corresponding menuItem
+        val menuItem: MenuItem = navView.menu.getItem(0)
+        menuItem.isChecked = true
+
+        navView.setOnNavigationItemSelectedListener(onNavigationItemSelectedListener)
+
+        setupRecycleView()
+    }
+}
```
Comment (chrisjwelly) [inline app/src/main/java/com/zhengcode/detask/DashboardTraitsActivity.kt]:
```suggestion
}

```

File: app/src/main/java/com/zhengcode/detask/DashboardTraitsAdapter.kt [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+        }
+
+        fun setData(traitStub: TraitStub, pos: Int) {
+            itemView.txvTitle.text = traitStub.name
+
+            this.currentTraitStub = traitStub
+            this.currentPosition = pos
+        }
+    }
+}
```
Comment (chrisjwelly) [inline app/src/main/java/com/zhengcode/detask/DashboardTraitsAdapter.kt]:
```suggestion
}

```

**Review by chrisjwelly — COMMENTED** [review]
Just adding some comments so that I can change the newline directly

**Review by BenFheng — APPROVED** [review]

**Pull request merged**